lib/modules/header-widgets/index.js

```diff
@@ -129,6 +153,18 @@ module.exports = {
                     widget.imageId = imageId;
                     widget.formattedImageStyles = styleSchema.format(imageId, widget.imageStyles);
                 }
+                
```

Comment:
Should this part be commented, or is it easily understandable?

Reply:
For me it's clear. Maybe rename it too manuallyIdeaCount? but I'm not sure...

Reply:
I've kept the name for now but added a comment about a possible caveat with the data.ideas.length being greater than or equal to 1000